content/en/platform/corda/1.6/cenm/release-notes.md

```diff
@@ -14,6 +14,23 @@ title: Release notes
 
 # Corda Enterprise Network Manager release notes
 
+## Corda Enterprise Network Manager 1.6.1
+
+### New features and enhancements
+
+* `sshdHost` can now be specified within the `shell {}` config block for the CENM services that support Shell.
+* Added a new `host` option in the `enmListener` configuration.
+* Added a new `host` option in the `adminListener` configuration.
```

Comment:
Added a new optional `host` parameter in the ... configuration

Reply:
updated